### PR TITLE
Geotiff is not a hard dependency of VXL

### PR DESCRIPTION
**Pull request by mattdawkins**

### CODE REVIEW

File: CMake/External_VXL.cmake [hunk 1]
```diff
@@ -20,11 +20,13 @@ add_package_dependency(
   )
 
 # libgeotiff
-add_package_dependency(
-  PACKAGE VXL
-  PACKAGE_DEPENDENCY libgeotiff
-  PACKAGE_DEPENDENCY_ALIAS GEOTIFF
-  )
```
Comment (dstoup) [inline CMake/External_VXL.cmake]: I think you can enable the same thing by using the OPTIONAL option in add_package_dependency, right?

Comment (mattdawkins) [reply CMake/External_VXL.cmake]: I'm not sure on the exact particulars of the script, ideally though we want to force geotiffs use if it is turned on at least, does the script do that?

**Comment by dstoup** [comment]
Yes. OPTIONAL should enable it if fletch builds it or it's found on the system but not error out if geotiff wasn't found.

**Comment by dstoup** [comment]
Since this branch is a good idea but you haven't fixed it, I am closing. There is a new branch which does this correctly. Have a look at #469

**Pull request closed without merging**